### ISSUE TITLE
Initial build error: CMake Error at CMake/GetGitRevisionDescription.cmake:61 (file):  file RELATIVE_PATH must be passed a full path to the file:

### ISSUE THREAD

**Issue by andrew-phi** [tdlib/td#2092]
0. Have Ubuntu 17.10 x64
1. Download and unzip td-master to /tmp
2. Build:
```rm -rf build
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=/usr/local ..
cmake --build . --target install```

3. Get error:

-- Performing Test HAVE_CXX_FLAG_WMAYBE_UNINITIALIZED - Success
CMake Error at CMake/GetGitRevisionDescription.cmake:61 (file):
  file RELATIVE_PATH must be passed a full path to the file:
Call Stack (most recent call first):
  CMakeLists.txt:156 (get_git_head_revision)

It may have something to do with downloading ZIP instead of using git (on slow connections it is way better).

**Comment by andrew-phi** [comment]
Workaround:
comment out CMakeLists.txt:156:
#include(GetGitRevisionDescription)
#get_git_head_revision(TD_GIT_REFSPEC TD_GIT_COMMIT_HASH)
#message(STATUS "Git state: ${TD_GIT_COMMIT_HASH}")
as these don't do any useful sh!t out there.

Also, at the end install fails, if not performed with root priveleges:

Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/lib/libtdjson.so.1.8.5
CMake Error at cmake_install.cmake:47 (file):
  file INSTALL cannot copy file "/tmp/td-master/build/libtdjson.so.1.8.5" to
  "/usr/local/lib/libtdjson.so.1.8.5".


Makefile:61: recipe for target 'install' failed
make: *** [install] Error 1

**Comment by levlam** [comment]
The correct fix has been implemented in da9f8078b835037b8e4aad00521d025b8092a109.
